server/models/appt.model.ts

```diff
@@ -1,33 +1,26 @@
-import { Document, Model, model, Schema } from "mongoose";
-
-export interface IAppointment extends Document {
-    name: string;
-    specialty: string;
-    time: string;
-    location: string;
-    notes: string;
-}
+import { Model, model, Schema } from "mongoose";
+import { IAppointment } from "../interfaces/appointment.interface";
 
 const appointmentSchema: Schema = new Schema({
```

Comment:
do we want to add a `rating` value? Simple numeric value that can be returned depending on the user selection via some sort of slider interaction.

Reply:
this is going to be put off via issue #32 .